test/Templates.Test/EmptyWebTemplateTest.cs

```diff
@@ -9,10 +13,16 @@ public EmptyWebTemplateTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Theory]
-        [InlineData(null)]
-        [InlineData("net461")]
-        public void EmptyWebTemplate_Works(string targetFrameworkOverride)
+        [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
+        public void EmptyWebTemplate_Works_NetFramework()
+            => EmptyWebTemplateImpl("net461");
```

Comment:
Are these changes just for stylistic preference, or was it essential for some reason? Just asking to make sure that if it is essential, I follow the necessary pattern in the future.

Reply:
It was essential to make the test project pass on Linux/macOS. This was the easiest way to skip only those tests that require .NET Framework. It's a pattern we've used in other places in our code. [Here's one example](https://github.com/aspnet/FileSystem/blob/cb0527b33185ca9e4ae05704dd9b83a4d16c2501/test/FS.Physical.Tests/PhysicalFileProviderTests.cs#L42-L70).